pyproject.toml

```diff
@@ -45,7 +45,8 @@ Issues = "https://github.com/opendatahub-io/vllm_tgis_adapter/issues"
 Source = "https://github.com/opendatahub-io/vllm_tgis_adapter"
 
 [project.scripts]
-grpc_healthcheck = "vllm_tgis_adapter.healthcheck:cli"
+grpc-healthcheck = "vllm_tgis_adapter.healthcheck:cli"
+vllm = "vllm_tgis_adapter.tgis_utils.scripts:cli"
```

Comment:
won't this over-write the original `vllm` CLI?

Reply:
I was wondering, should I use something like `tgis` instead since we're treating these as their own?

Reply:
Looking at this review comment from Nick, Ideally they should all be part of `vllm` itself
https://github.com/IBM/vllm/pull/16#pullrequestreview-1993895084. 

If we can't think of a way to override the name, I'm not sure if `tgis` makes sense though, might confuse people? The commands have nothing to do with `tgis`, they're just ported over from `tgis` which wouldn't make sense 🤔

Reply:
Also we should totally add a subcommand that will start the server using this adapter

Reply:
Right now having to type `python3 -m vllm_tgis_adapter` is a pain

Reply:
We can add a `vllm_tgis_adapter` cli entrypoint if required. If you want to save a character you can skip the `3` in `python3` though 😉

Reply:
Yeah if we're not able to extend the original `vllm` cli then an adapter cli would be great